fares-data-build-tool/fdbt-site
# Implement express sessions

# Description

This PR introduces express sessions middleware to the site with dynamodb used to store the sessions created. This will allow a user's session to persist without the extensive use of cookies we currently have.

Related PRs:
-   `fdbt-dev` repo - https://github.com/fares-data-build-tool/fdbt-dev/pull/13
-   `fdbt-aws` repo - https://github.com/fares-data-build-tool/fdbt-aws/pull/14

# Testing instructions

-   Pull down this branch and the corresponding branch (of the same name - `feature/TFN-549_setup_express_sessions`) in the `fdbt-dev` repo.
-   Run `make` in the `fdbt-dev` repo to spin up the site and localstack infrastructure.
-   After navigating through any portion of the site on any user journey, you can check that your session is stored in a local dynamodb table by running `awslocal dynamodb scan --table-name sessions`.

# Type of change


# Checklist:
